### ISSUE TITLE
squashfs unpacker is broken

### ISSUE THREAD

**Issue by weidenba** [fkie-cad/fact_extractor#5]
`/usr/bin/fakeroot: 175: /usr/bin/fakeroot: /opt/app/fact extractor/plugins/unpacking/squashFS/code/../bin/unsquashfs4-avm-be: not found`

any avm firmware should reproduce this

**Comment by dorpvom** [comment]
I can verify this. The tools are not compiled:
```
Makefile:90: *** Running makefile as root is prohibited! Please build Freetz as normal user.  Stop.
cp: cannot stat 'tools/unsquashfs4-avm-be': No such file or directory
cp: cannot stat 'tools/unsquashfs4-avm-le': No such file or directory
cp: cannot stat 'tools/unsquashfs3-multi': No such file or directory
```
Let's find out how we can get around this ...

**Comment by weidenba** [comment]
fixed in #8
